GenericDataList.c

```diff
@@ -0,0 +1,89 @@
+/*
+htop - GenericDataList.h
+(C) 2022 Sohaib Mohammed
+(C) 2022 htop dev team
+(C) 2022 Red Hat, Inc.
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "GenericDataList.h"
+
+#include "GenericData.h"
+#include "Hashtable.h"
+#include "Object.h"
+#include "Vector.h"
```

Comment:
```suggestion
#include "Object.h"
```

No need to repeat headers already present in the header file.